docker/package/wizard_structure.py

```diff
@@ -190,12 +190,15 @@ def get_proc_output(cmd):
     else:
         return subprocess.run(shlex.split(cmd), capture_output=True)
 
+
 def show_systemd_service(service_name):
```

Comment:
```suggestion
        self.config["node_rpc_addr"] = baking_env.get(
            "NODE_RPC_SCHEME",
            "http") + "://" + baking_env.get(
                "NODE_RPC_ADDR",
                "localhost:8732"
            )
```
Otherwise `tezos-client` fails when using `--endpoint` option